test_task1.py

```diff
@@ -0,0 +1,49 @@
+"""
+Tests for assignment1 and task1.
+"""
+
+import unittest
+import task1
+
+
+class StringManipulationTest(unittest.TestCase):
+    """
+    Tests for string manipulation task.
```

Comment:
Take a look at `ddt` https://ddt.readthedocs.io/en/latest/example.html
moving forward use this package for running test case with multiple inputs